includes/Modules/Analytics.php

```diff
@@ -589,6 +585,43 @@ protected function get_datapoint_services() {
 		);
 	}
 
+	/**
+	 * Gets map of datapoint to definition data for each.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array Map of datapoints to their definitions.
+	 */
+	protected function get_datapoint_definitions() {
+		$map = parent::get_datapoint_definitions();
```

Comment:
This is okay for now, but we should do a follow-up issue where all datapoints are migrated here (also for all other modules, and `get_datapoint_services()` should be deprecated.